geode-core/src/main/java/org/apache/geode/distributed/internal/AbstractDistributionConfig.java

```diff
@@ -1317,7 +1317,7 @@ static Class _getAttributeType(String attName) {
         "Specifies the address on which the Redis API for Geode is listening. If set to the empty string or this property is not specified, localhost is requested from the operating system.");
     m.put(REDIS_ENABLED,
         "When the default value of false, the Redis API for Geode is not available.  Set to true to enable the Redis API for Geode.");
-    m.put(REDIS_PASSWORD,
+    m.put(REDIS_USERNAME,
         "Specifies the password that the server uses when a client attempts to authenticate. The default is none and no authentication will be required.");
```

Comment:
I think you need to change the description here as well - it still refers to a password.

Reply:
Done